Bottom: a7714b2e293cb3631a9afde76baed99f451d1bea
Top:    56caa5a6fbed6dee87400c255ded17a7dad0cde7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:54:58 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
